core/src/main/java/io/kestra/core/repositories/MetricRepositoryInterface.java

```diff
@@ -0,0 +1,16 @@
+package io.kestra.core.repositories;
+
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.models.executions.MetricEntry;
+
+import java.util.List;
+
+public interface MetricRepositoryInterface extends SaveRepositoryInterface<MetricEntry> {
+    List<MetricEntry> findByExecutionId(String id);
```

Comment:
I would prefer a standard `ArrayListTotal<MetricEntry> find();` with multiple parameter and will avoid the lack of pagination that have log (and lead to freeze of the UI)

Reply:
I was hesitant as I didn't think they will be a lot of metrics (usually tens, rarely hundreds) and didn't want to change the existing UI that didn't have pagination.
Metrics are like outputs and there is no pagination on outputs.

Reply:
Done, Yann will do the UI part later

Reply:
Yann did the UI part, it's now OK